examples/dynamo/simple_int8_ptq.py

```diff
@@ -0,0 +1,46 @@
+import modelopt.torch.quantization as mtq
+import torch
+import torch_tensorrt as torchtrt
+from modelopt.torch.quantization.utils import export_torch_mode
+
+
+class SimpleNetwork(torch.nn.Module):
```

Comment:
We can actually use the existing VGG16 example for FP8 in `examples/dynamo` which measures FP8 accuracy on VGG model. You can use the same example and add section to it saying for INT8, we can modify the quant_cfg parameter.  cc: @narendasan  what do you think ?